vhdbuilder/release-notes/AKSAzureLinux/gen1/latest.txt

```diff
@@ -0,0 +1,702 @@
+iw-5.9-1.cm2.x86_64
+wireless-regdb-2022.08.12-1.cm2.noarch
+gpg-pubkey-3135ce90-5e6fda74
+gpg-pubkey-be1229cf-5631588c
+inotify-tools-3.22.1.0-1.cm2.x86_64
+fuse-2.9.7-10.cm2.x86_64
+boost-1.76.0-3.cm2.x86_64
+blobfuse-1.4.5-11.cm2.x86_64
+python3-dbus-1.2.16-3.cm2.x86_64
+python3-dateutil-2.7.3-5.cm2.noarch
+python3-dnf-plugins-core-4.0.24-3.cm2.noarch
+dnf-plugins-core-4.0.24-3.cm2.noarch
+dnf-utils-4.0.24-3.cm2.noarch
+check-restart-1.0.0-2.cm2.x86_64
+keyutils-1.6.1-1.cm2.x86_64
+cifs-utils-6.14-2.cm2.x86_64
+libnfnetlink-1.0.1-5.cm2.x86_64
+libnetfilter_conntrack-1.0.8-1.cm2.x86_64
+libnetfilter_queue-1.0.5-1.cm2.x86_64
+libnetfilter_cttimeout-1.0.0-5.cm2.x86_64
+libnetfilter_cthelper-1.0.0-5.cm2.x86_64
+conntrack-tools-1.4.5-7.cm2.x86_64
+dnf-automatic-4.8.0-2.cm2.noarch
+ebtables-legacy-2.0.11-6.cm2.x86_64
+apr-1.7.2-1.cm2.x86_64
+apr-util-1.6.3-1.cm2.x86_64
+utf8proc-2.6.1-2.cm2.x86_64
+libserf-1.3.9-8.cm2.x86_64
+subversion-1.14.2-1.cm2.x86_64
+subversion-perl-1.14.2-1.cm2.x86_64
+perl-YAML-1.30-2.cm2.noarch
+perl-DBI-1.643-2.cm2.x86_64
+perl-CGI-4.54-3.cm2.noarch
+less-590-2.cm2.x86_64
+git-2.33.8-1.cm2.x86_64
+iotop-0.6-10.cm2.noarch
+ipset-7.15-2.cm2.x86_64
+oniguruma-6.9.7.1-1.cm2.x86_64
+jq-1.6-1.cm2.x86_64
+kernel-devel-5.15.126.1-1.cm2.x86_64
+libtirpc-1.3.3-1.cm2.x86_64
+lsof-4.94.0-1.cm2.x86_64
+libpcap-1.10.1-1.cm2.x86_64
+nmap-ncat-7.93-1.cm2.x86_64
+rpcbind-1.2.5-5.cm2.x86_64
+libnfsidmap-2.5.4-2.cm2.x86_64
+libevent-2.1.12-1.cm2.x86_64
+nfs-utils-2.5.4-2.cm2.x86_64
+pigz-2.6-2.cm2.x86_64
+psmisc-23.4-1.cm2.x86_64
+socat-1.7.4.3-1.cm2.x86_64
+sysstat-12.7.1-2.cm2.x86_64
+traceroute-2.1.0-6.cm2.x86_64
+zip-3.0-5.cm2.x86_64
+libapparmor-3.0.4-1.cm2.x86_64
+apparmor-parser-3.0.4-1.cm2.x86_64
+fuse3-libs-3.10.5-2.cm2.x86_64
+fuse-common-3.10.5-2.cm2.x86_64
+fuse3-3.10.5-2.cm2.x86_64
+blobfuse2-2.0.5-2.cm2.x86_64
+libnftnl-1.2.1-1.cm2.x86_64
+jansson-2.14-1.cm2.x86_64
+nftables-1.0.1-1.cm2.x86_64
+moby-runc-1.1.9-1.cm2.x86_64
+moby-containerd-1.6.22-1.cm2.x86_64
+llvm-12.0.1-7.cm2.x86_64
+binutils-devel-2.37-5.cm2.x86_64
+binutils-2.37-5.cm2.x86_64
+compiler-rt-12.0.1-1.cm2.x86_64
+clang-libs-12.0.1-4.cm2.x86_64
+bcc-0.27.0-1.cm2.x86_64
+clang-12.0.1-4.cm2.x86_64
+libbpf-1.0.1-1.cm2.x86_64
+bpftrace-0.16.0-1.cm2.x86_64
+python3-bcc-0.27.0-1.cm2.x86_64
+bcc-tools-0.27.0-1.cm2.x86_64
+bcc-examples-0.27.0-1.cm2.x86_64
+=== Installed Packages End
+Disk usage:
+Filesystem      Size  Used Avail Use% Mounted on
+devtmpfs        4.0M     0  4.0M   0% /dev
+tmpfs           3.3G     0  3.3G   0% /dev/shm
+tmpfs           1.4G  8.8M  1.3G   1% /run
+/dev/sdb3        29G   21G  7.0G  75% /
+tmpfs           3.3G  4.0K  3.3G   1% /tmp
+/dev/sdb2       459M   35M  395M   9% /boot
+/dev/sda1        14G   28K   13G   1% /mnt
+tmpfs           668M     0  668M   0% /run/user/1000
+Using kernel:
+Linux version 5.15.126.1-1.cm2 (root@CBL-Mariner) (gcc (GCC) 11.2.0, GNU ld (GNU Binutils) 2.37) #1 SMP Wed Aug 23 01:21:36 UTC 2023
```

Comment:
@hbeberman just want to make sure that this should still say CBL Mariner, and all the names that follow in the release notes.

Reply:
Yeah it's expected that this still says CBLMariner.